### PR TITLE
Releases DataPrepper 2.10.2

### LINKED ISSUE #3487: [RELEASE] Release Data Prepper 2.10.2
### Is your feature request related to a problem? Please describe

We have a patch release of Data Prepper that we will release on Monday Dec 13, 2024

### Describe the solution you'd like to see

We will provide a PR with the downloads updates.

### Describe alternatives you've considered

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by kkondaka**
### Description
Releases DataPrepper 2.10.2
 
### Issues Resolved
Resolves #3487

### Check List
- [ X] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

File: _versions/2021-10-05-opensearch-1.1.0.markdown [hunk 1]
```diff
@@ -92,4 +92,4 @@ pretty:
     opensearch-dashboards-min: 'OpenSearch Dashboards Minimum'
 
 ---
-OpenSearch is open source software that uses the Apache License version 2 (ALv2). ALv2 grants you well-understood usage rights; you can use, modify, extend, embed, monetize, resell, and offer OpenSearch as part of your products and services. The source for the entire project is available on [GitHub](https://github.com/opensearch-project/) and you're welcome to build from source for customized deployments. Downloadable artifacts for OpenSearch and OpenSearch Dashboards include plugins and tools, ready for you to use with minimal configuration.
\ No newline at end of file
+OpenSearch is open source software that uses the Apache License version 2 (ALv2). ALv2 grants you well-understood usage rights; you can use, modify, extend, embed, monetize, resell, and offer OpenSearch as part of your products and services. The source for the entire project is available on [GitHub](https://github.com/opensearch-project/) and you're welcome to build from source for customized deployments. Downloadable artifacts for OpenSearch and OpenSearch Dashboards include plugins and tools, ready for you to use with minimal configuration.
```
Comment (dlvenable) [inline _versions/2021-10-05-opensearch-1.1.0.markdown:95]: Did you mean to edit this section? I see a few changes related to it.

Comment (kkondaka) [reply _versions/2021-10-05-opensearch-1.1.0.markdown:95]: No. Looks like that's a mistake. Let me check.

Comment (kkondaka) [reply _versions/2021-10-05-opensearch-1.1.0.markdown:95]: Looks like it's just a "newline" at the end of the line. Looks like my editor added a newline at the end. Is that a big deal?

**Review by dlvenable — APPROVED** [review]

**Comment by krisfreedain** [comment]
@kkondaka @dlvenable - pulled this local and it threw this error:

Error: YAML Exception reading /temp-kkondaka-dp-rel-2-10-2-project-website/_versions/2024-06-25-opensearch-2.15.0.markdown: (<unknown>): mapping values are not allowed in this context at line 19 column 16
      Sort warning: 'version' not defined in _versions/2024-06-25-opensearch-2.15.0.markdown
      
Wondering if you have an extra character in there. Can you check?

**Comment by krisfreedain** [comment]
nice - thanks for the quick fix @nateynateynate

**Review by krisfreedain — APPROVED** [review]

**Pull request merged**